lib/build.js

```diff
@@ -92,7 +92,8 @@ function getBlockedByIds(pipeline, job) {
                                 return [];
                             }
 
-                            return p.getJobs();
+                            // Without PR jobs
+                            return p.getJobs({ type: 'pipeline' });
```

Comment:
if current job is a PR job, should this return jobs pertaining to that PR ?

lets say something like 
```
if (current job is a PR job)
   return p.getJobs({ type: 'pr', prNum: prNum });
else  
   return p.getJobs({ type: 'pipeline' });
``` 

I think we don't support `p.getJobs({ type: 'pr', prNum: prNum });` right now. Idea behind PR https://github.com/screwdriver-cd/models/pull/509/files is also similar, limit the jobs being fetched for a PR, ideally it should be able to do `p.getJobs({ type: 'pr', prNum: prNum });`

Reply:
I think that works great for line 69. This may take some time to implement.

Line 96 is the process to determine if it is blocked by a remote pipeline jobs. Since the jobs retrieved here are from remote pipelines, the fact that the PR number is the same is not important for identification purposes. It is either blocked only on commit events, as in this fix, or it is not blocked at all.

Reply:
thanks for the clarification